welkin/models/calendar.py

```diff
@@ -174,20 +160,7 @@ def update(self, **kwargs):
 class WorkerHours(Collection):
     resource = WorkHours
 
-    def get(
-        self,
-        from_date: datetime,
-        to_date: datetime,
-        ids: list = None,
-        timezone: str = None,
-    ):
-        params = {
-            "psmIds": ids,
-            "from": from_date,
-            "to": to_date,
-            "timezone": timezone,
-        }
-
+    def get(self, *args, **kwargs):
         return super().get(
-            f"{self._client.instance}/calendar/work-hours", params=params
+            f"{self._client.instance}/calendar/work-hours", *args, **kwargs
```

Comment:
Sorry if my comment wasn't clear, I just meant in addition to what you had already with the payload set up, this method should accept args and kwargs